api/composer.json

```diff
@@ -27,7 +27,9 @@
         "symfony/serializer": "6.4.*",
         "symfony/twig-bundle": "6.4.*",
         "symfony/validator": "6.4.*",
-        "symfony/yaml": "6.4.*"
+        "symfony/yaml": "6.4.*",
+        "twig/extra-bundle": "^2.12|^3.0",
+        "twig/twig": "^2.12|^3.0"
     },
     "require-dev": {
```

Comment:
schema-generator, phpunit-bridge, browser-ki, css-selectror and maker bundle can be safely removed.